diff --git a/node_modules/react-native-pell-rich-editor/src/RichEditor.js b/node_modules/react-native-pell-rich-editor/src/RichEditor.js
index 14bea22..c263f75 100755
--- a/node_modules/react-native-pell-rich-editor/src/RichEditor.js
+++ b/node_modules/react-native-pell-rich-editor/src/RichEditor.js
@@ -266,7 +266,7 @@ export default class RichTextEditor extends Component {
           {...rest}
           ref={that.setRef}
           onMessage={that.onMessage}
-          originWhitelist={['*']}
+          originWhitelist={Array.isArray(rest.originWhitelist) ? rest.originWhitelist : ['*']}
           dataDetectorTypes={'none'}
           domStorageEnabled={false}
           bounces={false}
